c7n/resources/athena.py

```diff
@@ -0,0 +1,16 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.manager import resources
+from c7n import query
+
+
+@resources.register("athena-named-query")
```

Comment:
This seems to be looking at saved athena queries. @kapilt does it make sense to have this as a resource type? Looks like they don't even support tagging. But honestly, there's no harm

Reply:
yeah.. its not entirely clear to me on use case, I had a request for a bunch of resources, and this got rolled into the mix. it feels like low cardinality versus data plane, so I don't see the harm. would like to followup with other Athena resources workgroup